src/features/user/Settings/ApiKey/ApiKeyForm.tsx

```diff
@@ -76,19 +86,10 @@ export default function ApiKey({}: any) {
   }, [token, contextLoaded]);
 
   return (
-    <div className="profilePage" style={{ backgroundColor: '#fff' }}>
-      <p className={'profilePageTitle'}> {t('apiKey')}</p>
-      <div className={styles.apiPage}>
-        <p className={styles.deleteModalContent}>
-          {t('me:apiKeyMessage1')}
-          {/* <br /><br />
-                   {t('me:apiKeyMessage2')} */}
-          <br />
-          <br />
-          {t('me:apiKeyMessage3')}
-        </p>
+    <StyledForm>
+      <div className="inputContainer">
         <div className={styles.apiKeyContainer}>
```

Comment:
Would it be possible to replace the `<div className={styles.apiKeyContainer}>` with an `<InlineFormDisplayGroup>`? 

You may need to make some adjustments (e.g. to incorporate the hide/view key button - this could be useful https://mui.com/material-ui/react-text-field/#input-adornments)

The reason I'm suggesting this is for consistency - I can see some slight differences in the 2 places. One example is the copy button which has slightly different dimensions inspite of using a common component.

Remember to remove unused styles from ApiKey.module.scss if this is done.

Reply:
Made a minor change to pass `type=other` to `InlineFormDisplayGroup` as it does not contain only text fields. This vertically centers the children contained within `InlineFormDisplayGroup`.